core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4628,6 +4647,9 @@ protected function listView()
 			$firstOrderBy = $this->firstOrderBy;
 		}
 
+		// Check the default labels (see #509)
+		$labelNew = $GLOBALS['TL_LANG'][$this->strTable]['new'] ?? $GLOBALS['TL_LANG']['DCA']['new'];
```

Comment:
If you gast this to an array, then the value would become the button if there is only one. No title then, but I guess thats fine?